client/src/Pos/Client/Txp/Util.hs

```diff
@@ -557,7 +557,7 @@ createTx
     -> m (Either TxError TxWithSpendings)
 createTx pendingTx utxo ss outputs addrData =
     createGenericTxSingle pendingTx (\i o -> Right $ makePubKeyTx ss i o)
-    OptimizeForSecurity utxo outputs addrData
+    OptimizeForHighThroughput utxo outputs addrData
```

Comment:
Is this used only in benchmarking, or also in the wallet?

Reply:
It is only used in `auxx`. So it is not used in the wallet.

Reply:
Are you sure? It is used in `submitTx` in client/src/Pos/Client/Txp/Network.hs, and that function is used quite a lot.

Reply:
@kantp 
`submitTx` is removed by this PR because it was not used in the code.
If you see any `submitTx` in the `develop` branch it is because it is used also as a parameter of a function with the same name.

Reply:
Ah, I see! Sorry about the confusion.